### PR TITLE
100% test coverage for client/__init__.py

### PR DESCRIPTION
**Pull request by ryanlerch**
Signed-off-by: Ryan Lerch <rlerch@redhat.com>

### CODE REVIEW

File: bodhi/client/__init__.py
Comment (bowlofeggs) [inline bodhi/client/__init__.py:691]: At first this made me nervous, but I realized that we already have code that handles this in the bindings so this would never happen:

https://github.com/fedora-infra/bodhi/blob/2.11.0/bodhi/client/bindings.py#L92-L101

File: bodhi/tests/client/test___init__.py
Comment (bowlofeggs) [inline bodhi/tests/client/test___init__.py]: This is probably a debug artifact - can you remove it before merging?

File: bodhi/tests/client/test___init__.py
Comment (bowlofeggs) [inline bodhi/tests/client/test___init__.py]: Same here.

**Review by bowlofeggs — APPROVED** [review]
Looks good! Can you drop the two maxDiff's before merging? Thanks for the test coverage, it's fantastic!

File: bodhi/client/__init__.py
Comment (ryanlerch) [reply bodhi/client/__init__.py:691]: Sorry! i meant to mention this in the PR description. :)

I was a little hard to write a test for something that we would never hit :D

**Comment by ryanlerch** [comment]
this is now failing CI after rebasing due to failing tests in develop. we can merge this after #1837

**Comment by bowlofeggs** [comment]
jenkies test

**Review by bowlofeggs — APPROVED** [review]
LGTM!

**Pull request merged**
